Packs/Legacy/Widgets/CHANGELOG.md

```diff
@@ -0,0 +1,230 @@
+## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```

Comment:
what

Reply:
it's the release_notes_clear.py script...